man/man8/zfs-userspace.8

```diff
@@ -44,15 +44,15 @@
 .Oo Fl s Ar field Oc Ns ...
 .Oo Fl S Ar field Oc Ns ...
 .Oo Fl t Ar type Ns Oo , Ns Ar type Oc Ns ... Oc
-.Ar filesystem Ns | Ns Ar snapshot
+.Ar filesystem Ns | Ns Ar snapshot | Ns Ar path
```

Comment:
I think these all need another Ns before the bar:
```suggestion
.Ar filesystem Ns | Ns Ar snapshot Ns | Ns Ar path
```

Reply:
fixed